scss/6-components/_on-this-page.scss

```diff
@@ -8,29 +8,15 @@
   padding: $cads-spacing-4;
   margin: 0 0 $cads-spacing-5 0;
 
-  &__title {
+  .cads-on-this-page__title {
     @extend %cads-h4;
 
     margin: 0 0 $cads-spacing-4 0;
     color: $cads-language__secondary-text-colour;
   }
 
-  &__list {
+  .cads-on-this-page__list {
     @extend %cads-list-no-bullet;
-
-    padding: 0;
-    margin: 0;
-    list-style-type: none;
-  }
-
-  &__list-item {
-    position: relative;
-
-    @include cads-media-breakpoint-up(md) {
-      -webkit-column-break-inside: avoid;
-      page-break-inside: avoid;
-      break-inside: avoid;
-    }
```

Comment:
This was the main issue, combined with `columns: 1` was causing a miscalculation depending on the zoom level. Ironically these styles exist to workaround a _different_ css columns bug, making sure items don't wrap across two columns. Moves these styles to be scoped to the `--columns` variant.